course/test/src/Ant_TEST.cc

```diff
@@ -0,0 +1,129 @@
+#include <cmath>
+#include <sstream>
+#include <string>
+#include <memory>
+
+#include "gtest/gtest.h"
+
+#include "ant.h"
+#include "cell_mock.h"
+
+namespace ekumen {
+namespace simulation {
+namespace test {
+namespace {
+
+GTEST_TEST(AntTest, OccupiedCellOnInitialization) {
+  std::shared_ptr<Cell> cell(new Cell());
+  Ant ant(cell);
+  EXPECT_FALSE(cell->IsFree());
+}
+
+GTEST_TEST(AntTest, CallbackAssignedCorrectly) {
+  std::shared_ptr<Cell> cell(new Cell());
+  Ant ant = Ant(cell);
+  bool lambda_called = false;
```

Comment:
Prefer braces initialization `bool lambda_called{false};`
See `A tour of cpp` pages 6-7:
```
The = form is traditional and dates back to C, but if in doubt, use the general {} -list form. If nothing
else, it saves you from conversions that lose information:
  int i1 = 7.2;  // i1 becomes 7 (surprise?)
  int i2 {7.2};  // error : floating-point to integer conversion
  int i3 = {7.2};  // error : floating-point to integer conversion (the = is redundant)
Unfortunately, conversions that lose information, narrowing conversions, such as double to int and
int to char are allowed and implicitly applied. The problems caused by implicit narrowing conver-
sions is a price paid for C compatibility (§14.3).
```

Reply:
Done.